neuromation/cli/formatters/jobs.py

```diff
@@ -78,6 +78,8 @@ def __call__(self, job_status: JobDescription) -> str:
         result: str = f"Job: {job_status.id}\n"
         if job_status.name:
             result += f"Name: {job_status.name}\n"
+        if job_status.tags:
+            result += f"Tags: {', '.join(job_status.tags)}\n"
```

Comment:
Maximum number of the job's tags is `256`, max length of each tag is `256`. I don't think it's a problem to have a very-long line in the worst case, but if anyone has concerns, we could think how to separate this value to multiple lines.

Reply:
It is easy.
```python
text = "Tags: " + ", ".join(job_status.tags)
if root.tty:
    width = root.get_terminal_size[0]
    text = textwrap.fill(text, width=width, subsequent_indent="      ", break_long_words=False)
result += text +"\n"
```

Reply:
Thank you. This is the issue not to forget: https://github.com/neuromation/platform-client-python/issues/1408